modules/log6x-oc-explain.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+:_mod-docs-content-type: CONCEPT
+[id="log6x-oc-explain_{context}"]
+
+= Using the `oc explain` command
+
+The `oc explain` command is an essential tool in the OpenShift CLI `oc` that provides detailed descriptions of the fields within Custom Resources (CRs). This command is invaluable for administrators and developers who are configuring or troubleshooting resources in an OpenShift cluster.
+
+== Resource Descriptions
+`oc explain` offers in-depth explanations of all fields associated with a specific object. This includes standard resources like pods and services, as well as more complex entities like statefulsets and custom resources defined by Operators.
+
+For example, to view the documentation for the `outputs` and `options` fields of the `ClusterLogForwarder` custom resource, you can use:
+
+[source,terminal]
+----
+$ oc explain clusterlogforwarders.observability.openshift.io.spec.outputs
+$ oc explain clusterlogforwarders.observability.openshift.io.spec.outputs.options
```

Comment:
This should probably fail? in Logging 5 was not available `spec.outputs.options` and not able to see it in https://github.com/openshift/cluster-logging-operator/blob/master/config/crd/bases/observability.openshift.io_clusterlogforwarders.yaml